cache.go

```diff
@@ -159,7 +159,7 @@ func (cache *CacheDriver) Touch(keyVal interface{}) error {
 		if entry.state != COMPUTING && entry.state != AVAILABLE {
```

Comment:
I had forgotten to comment that LazyRemove, Touch, and Contains are similar in structure. We could find some form of drying them. Perhaps by coding a general but private function that receives a lambda. Then we could set every public function (LazyRemove, Touch, and Contains) through a different lambda that implements the differences. Let me know if you want me to explain the idea. Also, let me know if you find another approach.